filebeat/harvester/reader/json.go

```diff
@@ -116,10 +116,12 @@ func transformNumbersArray(arr []interface{}) {
 
 // Next decodes JSON and returns the filled Line object.
 func (r *JSON) Next() (Message, error) {
-	reader, err := r.reader.Next()
+	message, err := r.reader.Next()
 	if err != nil {
-		return reader, err
+		return message, err
 	}
-	reader.Content, reader.Fields = r.decodeJSON(reader.Content)
-	return reader, nil
+	var fields = common.MapStr{}
```

Comment:
I think this should be `var fields common.MapStr` since you don't need to allocate a `MapStr` because `decodeJSON` will be returning one.